core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -43,28 +57,45 @@
-        <schema-based-editor *ngIf="editorIsShown && activeDataFormat === 'html'"
-                             [schema]="HTML_SCHEMA"
-                             [localValue]="activeWrittenTranslation"
-                             (localValueChange)="updateHtml($event)">
-        </schema-based-editor>
-        <schema-based-editor *ngIf="editorIsShown && activeDataFormat === 'unicode'"
-                             [schema]="UNICODE_SCHEMA"
-                             [localValue]="activeWrittenTranslation"
-                             (localValueChange)="updateHtml($event)">
-        </schema-based-editor>
-        <div *ngIf="editorIsShown && isSetOfStringDataFormat()" class="oppia-rules-to-translate-container">
-          <strong>Please add a translation per answer choice.</strong>
-          <p class="oppia-rules-translation-tip">
-            Because these answer choices will be matched against a learner's input, we want to be extra careful with the translations so as to not block a learner's progress in a lesson.
-          </p>
-          <schema-based-editor [schema]="SET_OF_STRINGS_SCHEMA"
+        <div class="oppia-editor-container"
+             [ngClass]="isTranslationExpanded ? 'oppia-container-expanded' : 'oppia-container-contracted'"
+             #translationContainer>
+          <schema-based-editor *ngIf="editorIsShown && activeDataFormat === 'html'"
+                               [schema]="HTML_SCHEMA"
+                               [localValue]="activeWrittenTranslation"
+                               (localValueChange)="updateHtml($event)"
+                               #oppiaRTE>
+          </schema-based-editor>
+          <schema-based-editor *ngIf="editorIsShown && activeDataFormat === 'unicode'"
+                               [schema]="UNICODE_SCHEMA"
                                [localValue]="activeWrittenTranslation"
-                               (localValueChange)="updateHtml($event)">
+                               (localValueChange)="updateHtml($event)"
+                               #oppiaRTE>
           </schema-based-editor>
+          <div *ngIf="editorIsShown && isSetOfStringDataFormat()"
+               class="oppia-rules-to-translate-container">
+            <strong>Please add a translation per answer choice.</strong>
+            <p class="oppia-rules-translation-tip">
+              Because these answer choices will be matched against a learner's input, we want to be extra careful with the translations so as to not block a learner's progress in a lesson.
+            </p>
+            <schema-based-editor [schema]="SET_OF_STRINGS_SCHEMA"
+                                 [localValue]="activeWrittenTranslation"
+                                 (localValueChange)="updateHtml($event)">
+            </schema-based-editor>
+          </div>
+        </div>
+        <div class="oppia-collapse-content"
+             (click)="toggleExpansionState('translation')"
```

Comment:
Are you able to reference the enum directly, i.e. `ExpansionTabType.TRANSLATION`, or does that not work because it's not available in the Angular component scope?

Reply:
Nope, referencing the enum in the html does't work. Perhaps we could use a variable with the enum value like: `contentTab: number = 0` and similarly for 'translation'?